lib/tunemygc/agent.rb

```diff
@@ -53,19 +52,12 @@ def spies
   end
 
   def recommendations
-    MUTEX.synchronize do
-      require "tunemygc/syncer"
-      syncer = TuneMyGc::Syncer.new
-      config = syncer.sync(snapshotter)
-    end
-  rescue Exception => e
-    log "Config recommendation error (#{e.message})"
+  # no op
   end
 
   extend self
 
   MUTEX.synchronize do
-    self.logger = Logger.new($stdout)
```

Comment:
There is mutex grabbed in logger that in certain conditions can't be grabbed which results in `stderr` being flooded with error messages (the sighandler thing I mentioned). This was tripped when the queue was full and it wanted to log it drops messages. 
Given I don't use the queue anymore, I _think_ I could undo that change. It "just felt" safer to not log to avoid (other?) possible situations like this one.